lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4635,7 +4636,6 @@ DefEnvironment('{minipage}[][][]{Dimension}', sub {
   sizer      => '#body',
   mode       => 'text',
   properties => sub { (
-      layout  => 'vertical',
```

Comment:
This was wrong; minipage does not (necessarily) lay out its contents vertically.

I'm also concerned about similar properties that snuck in on the various floats. Floats don't (necessarily) lay out contents vertically. (unless the contents don't fit).

What were we trying to accomplish here?

Reply:
the way the width gets computed is different when this property is set. 

For the example I understood, we don't want to add the width of a figure image with the width of a caption to get the width of the entire figure. As the figure is vertical, the width of widest component is what determines the overall width.

Maybe I overstepped with minipages and floats - though I recall floats worked rather well in my tests. Minipages were bumpy in arXiv whether I had this attribute or not, since there was a lot more going on there.

Reply:
Oh and the reason I needed some "correct" width was to determine if I need to auto-generate a line break, as I was trying to emulate the same subfigure grids that were originally created for the PDF output.

Reply:
Well, vertical means that *all* the children are stacked vertically, rather than fitting them to the current width, and wrapping when needed. So, of course, it affects both the width and height.

But, captions explains why floats/figures/tables have that property (I think your changes just set them more consistently). I think it's probably the wrong way to deal with captions in the long run; really should treat the caption as a block, but wrap the other components.  But that's tricker to set up.

However, for minipages, it's just wrong. And besides, minipages already have their width preset.